examples_tests/44.LevelCurveExtraction/main.cpp

```diff
@@ -66,29 +66,80 @@ int main()
     
     //copy the pipeline
     auto pipeline_cp = core::smart_refctd_ptr_static_cast<asset::ICPURenderpassIndependentPipeline>(mesh_raw->getMeshBuffer(0u)->getPipeline()->clone());
```

Comment:
u dont need full (deep) copy, you need to copy only the pipeline object and maybe its layout, so pass 1u as arg to copy() (then it will copy pipeline, layout and specialized shaders)